docs/change-log.md

```diff
@@ -2,6 +2,18 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update September 2nd, 2022
+
+- **Improved:** The UI for [container overrides](https://docs.humanitec.com/using-humanitec/work-with-workloads/manage-container-configuration#change-container-configuration) on the [Workload Details Page](https://docs.humanitec.com/reference/user-interface/workload-details-screen#container-panel) has been reworked. Command and argument overrides are now configured via a code editor.  
```

Comment:
> are now configured via a code editor.  

Maybe say that they can be defined directly as YAML?  Or do you think the code editor is more important?